debian/rules

```diff
@@ -28,7 +28,7 @@ include /usr/share/dpkg/default.mk
 #	-DCMAKE_LIBRARY_PATH=$(DEB_HOST_MULTIARCH)
 
 override_dh_auto_configure:
-	dh_auto_configure --
+	dh_auto_configure -- ${CONFIGURE_ARGS}
```

Comment:
The idea was to be able generate sonic-swss binaries with debug symbols enabled.

The usage will be as follows.
export CONFIGURE_ARGS=--enable-debug=yes
cd sonic-buildimage/src/sonic-swss
make